flatdata-cpp/test/MultiVectorTest.cpp

```diff
@@ -251,6 +277,20 @@ TEST( MultiVectorTest, IterateOneTypeElementsRandomlyPlaced )
     ASSERT_TRUE( has_c2 );
 }
 
+TEST( MultiVectorTest, FlushingWhileBuilding )
+{
+    auto view = create_view_with_enough_items_to_flush( );
+
+    for ( size_t i = 0; i < NUM_ITEM_TO_FLUSH; i++ )
+    {
+        auto iter = view.second.iterator< CStruct >( i );
+        ASSERT_TRUE( iter.valid( ) ) << "Index " << i;
+        ASSERT_EQ( i, ( *iter ).value ) << "Index " << i;
```

Comment:
Message: "Expected: " << i << " at index: " << i << " Actual: " << ...

Reply:
Not require (see above)